README.md

```diff
@@ -125,57 +185,58 @@ This project also uses ESLint and sass-lint to help find bugs and enforce code q
 
 ### Other
+
 - To switch between the DUS Classic version and Amazon Kendra enabled version, please follow a fresh deploy (either in a different region/ deleting the stack) and avoid updating the CloudFormation stack for the existing mode. Currently, DUS does not have the feature to seamlessly switch between the 2 modes.
-More info available in this [issue](https://github.com/awslabs/document-understanding-solution/issues/83) 
+  More info available in this [issue](https://github.com/awslabs/document-understanding-solution/issues/83)
 - Do NOT change the `cicd` in package.json. This field is for the deployment system to use in CodePipeline
 - Due to limitations of CodeCommit, you cannot use this deploy approach if you add a file to the solution that is above 6MB (for good measure, stay below 5MB)
 
-
 ## Cost
 
 - As you deploy this sample application, it creates different resources (Amazon S3 bucket, Amazon SQS Queue, Amazon DynamoDB table, Elasticsearch (and potenitally Amazon Kendra) clsuter(s) and AWS Lambda functions etc.). When you analyze documents, it calls different APIs (Amazon Textract , Amazon Comprehend and Amazon Comprehend Medical) in your AWS account. You will get charged for all the API calls made as part of the analysis as well as any AWS resources created as part of the deployment. To avoid any recurring charges, delete stack using "yarn destroy".
 
 - The CDK Toolkit stacks that are created during deploy of this solution are not destroyed when you tear down the solution stacks. If you want to remove these resources, delete the S3 bucket that contains `staging-bucket` in the name, and then delete the `CDKToolkit` stack.
 
 - You are responsible for the cost of the AWS services used while running this reference
-deployment. The solution consists of some resources that have to be paid by the hour/size
-such as Amazon Elasticsearch, Amazon Kendra and Amazon S3 while others are serverless technologies where
-costs are incurred depending on the number of requests.
-The approximate cost for the solution for 100 documents/day comes under $20/day for the Classic Mode and under $80/day for Kendra-Enabled Mode. For accurate and most up-to-date pricing information, refer [AWS Pricing](https://aws.amazon.com/pricing/)
-
+  deployment. The solution consists of some resources that have to be paid by the hour/size
+  such as Amazon Elasticsearch, Amazon Kendra and Amazon S3 while others are serverless technologies where
+  costs are incurred depending on the number of requests.
+  The approximate cost for the solution for 100 documents/day comes under $20/day for the Classic Mode and under $80/day for Kendra-Enabled Mode. For accurate and most up-to-date pricing information, refer [AWS Pricing](https://aws.amazon.com/pricing/)
 
 ## Delete demo application
 
-Development Deploy:
+1. Full CICD Deploy:
```

Comment:
Any reason we are calling it Full CICD deploy vs CICD deploy?

Reply:
nope haha. Updated.